### ISSUE TITLE
script Tags from storyblok components are not bundled properly when using astrojs > 2.9 with the new hoisted script optimization

### ISSUE THREAD

**Issue by wernerstrauch** [storyblok/storyblok-astro#380]
Script Tags (<script>) from storyblok components are not bundled properly when using astrojs >= 2.9 with the new hoisted script optimization.

AstroJS released a new featured combining script tags more efficiently:
https://astro.build/blog/astro-290/#hoisted-script-optimization

The problem is now that <script> Tags from Storyblok Components are not bundled with the hoisted script optimization.

This occurs only then you run bundle + preview.
In dev mode all script tags gets executed.

![@storyblokastro 2023-08-04 16-44-12](https://github.com/storyblok/storyblok-astro/assets/6968718/3e156db2-32cb-42c6-a007-6c76d944c745)

https://github.com/wernerstrauch/astrojs-storyblok-hoisted-issue

---

## Expected Behavior

BaseLayout.astro
```
<script> 
  console.log("Script from BaseLayout.astro is executed");
</script> 
```

Grid.astro
```
<script> 
  console.log("Script from Grid.astro is executed");
</script> 
```

Console Output:
```
Script from BaseLayout.astro is executed
Script from Grid.astro is executed
```

## Current Behavior

BaseLayout.astro
```
<script> 
  console.log("Script from BaseLayout.astro is executed");
</script> 
```

Grid.astro
```
<script> 
  console.log("Script from Grid.astro is executed");
</script> 
```

Console Output:
```
Script from BaseLayout.astro is executed
```

Console.log from Grid.astro is missing here

## Steps to Reproduce

1. Add <script> Tag to Storyblok Component
2. Run npm run build
3. Run npm run preview

**Comment by manuelschroederdev** [comment]
Hey @wernerstrauch, thanks for opening this issue, we'll look into it.

**Comment by manuelschroederdev** [comment]
This should be fixed with `[Astro 2.10.5](https://github.com/withastro/astro/releases/tag/astro%402.10.5)`. 🙂
